src/main/resources/application.yaml

```diff
@@ -144,3 +144,4 @@ task:
   check-jurisdiction-log-in:
     check-validity-duration: PT5M # only ensure this often that no log-in attempt is rejected by IDAM
 
+supported-jurisdictions: SSCS,BULKSCAN,DIVORCE,PROBATE,FINREM,CMC
```

Comment:
All these values are already present in application.yaml in service-config.services.service. Adding this new property makes sense only if we can configure a service but not to support it, otherwise it is just adding maintenance overhead.

Reply:
I agree.. but then, we will need a property for each service in AKS. Alternatively, we can just have 1 property for FPLA and remove it later (when it is live)